README.md

```diff
@@ -14,134 +20,450 @@ For suggestions on improvements,
-[Sync](https://docs.databricks.com/repos.html) this repository with
-[Databricks Repos](https://docs.databricks.com/repos.html) and run the `notebooks/databricks`
-notebook on a Databricks Cluster running version 11.0 or greater of the
-[Databricks Runtime](https://docs.databricks.com/runtime/dbr.html) or the
-[Databricks Runtime for Machine Learning](https://docs.databricks.com/runtime/mlruntime.html)
-with [workspace files support enabled](https://docs.databricks.com/repos.html#work-with-non-notebook-files-in-a-databricks-repo).
-
-**Note**: When making changes to pipelines on Databricks,
-it is recommended that you edit files on your local machine and
-use [dbx](https://docs.databricks.com/dev-tools/dbx.html) to sync them to Databricks Repos, as
-demonstrated [here](https://mlflow.org/docs/latest/pipelines.html#usage)
-
-**Note**: data profiles display in step cards are not visually compatible with dark theme.
-Please avoid using the dark theme if possible.
-
-### Accessing MLflow Pipeline Runs
-You can find MLflow Experiments and MLflow Runs created by the pipeline on the
-[Databricks ML Experiments page](https://docs.databricks.com/applications/machine-learning/experiments-page.html#experiments).
-
-## Development Environment -- Local machine
-### Jupyter
-1. Launch the Jupyter Notebook environment via the `jupyter notebook` command.
-2. Open and run the `notebooks/jupyter.ipynb` notebook in the Jupyter environment.
-
-**Note**: data profiles display in step cards are not visually compatible with dark theme.
-Please avoid using the dark theme if possible.
-
-### Command-Line Interface (CLI)
-
-First, enter the template root directory and set the profile via environment variable
+This is a visual overview of the MLflow Regression Pipeline's information flow.
+
+Model develompent consists of the following sequential steps:
 ```
-cd mlp-regression-template
+ingest -> split -> transform -> train -> evaluate -> register
 ```
+
+The batch scoring workflow consists of the following sequential steps:
 ```
-export MLFLOW_PIPELINES_PROFILE=local
+ingest -> predict
 ```
+A detailed reference for each step follows.
 
-Then, try running the
-following [MLflow Pipelines CLI](https://mlflow.org/docs/latest/cli.html#mlflow-pipelines)
-commands to get started.
-Note that the `--step` argument is optional.
-Pipeline commands without a `--step` specified act on the entire pipeline instead.
+ * [Reference](#reference)
+    + [Step artifacts](#step-artifacts)
+    + [Ingest step](#ingest-step)
+      - [Data](#data)
+    + [Split step](#split-step)
+    + [Transform step](#transform-step)
+    + [Train step](#train-step)
+    + [Evaluate step](#evaluate-step)
+    + [Register step](#register-step)
+    + [MLflow Tracking / Model Registry configuration](#mlflow-tracking--model-registry-configuration)
+    + [Metrics](#metrics)
+      - [Built-in metrics](#built-in-metrics)
+      - [Custom metrics](#custom-metrics)
+  * [Scoring](#scoring)
```

Comment:
`Scoring` -> `Batch Scoring` to be more specific. Is this row supposed to be aligned with `Reference`? May be better to align with `Model Development`

Reply:
Yeah I'll just move scoring under reference